workers/ohsome_quality_analyst/reports/multilevel_currentness/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+MultilevelCurrentness:
+  name: Multilevel Currentness
+  description: |
+    This report shows the currentness of four major Map Features (https://wiki.openstreetmap.org/wiki/Map_features): buildings, land-use/land-cover, points of interest and infrastructure.
+  label_description:
```

Comment:
I think these descriptions have been copied over from the mapping saturation indicator and needs adjustment

Reply:
I used the report here just for testing and thought that the new report belongs in a separate PR.  Or would you keep it in this PR?